src/main/java/seedu/lifeasier/model/tasks/TaskList.java

```diff
@@ -22,7 +23,8 @@
 
     private static Logger logger = Logger.getLogger(ShowNotesCommand.class.getName());
     protected static ArrayList<Task> taskList;
-    private int indexOfLastMatch;
+    private int displayIndexOfLastMatch = 0;
+    HashMap<Integer,Integer> map;
```

Comment:
Wow nice use of a hashmap. However, could the name be made more descriptive?

Reply:
Maybe something like `taskMap`